lib/chef-dk/policyfile_lock.rb

```diff
@@ -15,7 +16,7 @@
 # limitations under the License.
 #
 
-require 'digest/sha1'
+require 'digest/sha2'
```

Comment:
Also, I plan to change the way cookbook artifact IDs are generated in a similar way, but all that will be done before taking off the "beta" sticker (or else an upgrade path will be given).